web/modules/custom/vote_utility/translations/nv.po

```diff
@@ -13,214 +8,285 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\\n"
 
-#language selector
+# config.languages.languageName
 msgid "Navajo"
 msgstr "Diné"
 
+# config.languages.params.select_language
 msgid "Select language"
 msgstr "Saad choid77['88[7g77"
 
-#main
+# config.languages.params.Owner
 msgid "@sitename in language"
-msgstr "@sitename Diné"
+msgstr "@sitename in Diné"
 
+# config.languages.params.go_back
 msgid "Go back"
 msgstr "N1t'33'"
 
-msgid "Illustration of voter box with checked ballot being inserted to indicate voting action"
-msgstr "&'ii'n77[ b1 sits'aa y1zh7 si'1n7g77 biyi' asdzoh naashch'22'7g77 47 n7l47 o'oot'ah n7igo yaa y1[ti'go si'32go 1lyaa"
-
+# config.languages.params.last_updated
 msgid "Last updated:"
-msgstr "Ak44'di n1zk'33z:"
+msgstr "Ak44'di n1zk'33z: "
```

Comment:
remove empty space at end of string.

Reply:
removed tailing space at the end of string